### PR TITLE
Set port via cmd line argument and update README

### PR DESCRIPTION
**Pull request by ndoschek**
- Enable custom model server port via command line argument
- Update README.md

### CODE REVIEW

File: com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java [hunk 1]
```diff
@@ -33,6 +33,8 @@
 	private String workspaceRoot;
 	private String[] args;
 
+	private static final int DEFAULT_JAVALIN_PORT = 8081;
```
Comment (tortmayr) [inline com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java]: Should be a public constant

Comment (tortmayr) [inline com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java]: Should be a public constant

File: com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java [hunk 2]
```diff
@@ -57,7 +59,9 @@ public void start() {
 	}
 
 	protected void run() {
-		injector.getInstance(ModelServerStartup.class).boot(EntryPointType.REST, this.args);
+		int port = args.length > 0 ? Integer.parseInt(args[0].split("=")[1]) : DEFAULT_JAVALIN_PORT;
```
Comment (tortmayr) [inline com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java]: This causes an uncatched exceptions (`ArrayIndexOutOfBoundsException`, `NumberFormatException`) if the argument doesn't have the right format. (no `=` or the right-hand side of `=` cannot be cast to an integer)

We should catch this and print some information about invalid arguments and the correct usage.

In addition, the --port prefix is currently not checked. IMHO it should either be checked if the argument starts with "--port" or the prefix should be omitted.

**Review by tortmayr — CHANGES_REQUESTED** [review]
Thanks for improving the readme!
I noticed a minor issue with the port parameter parsing. Other than that LGTM.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,35 @@
...
-## Build and run the standalone jar
+To build the modelserver as standalone JAR execute the following maven goal in the root directory:
 ```bash
 mvn clean install -Pfatjar
+```
+
+
+## Run the modelserver
+### Run the modelserver in IDE
+To run the example modelserver within an IDE, run the main method of [ExampleServerLauncher.java](https://github.com/eclipsesource/modelserver/blob/master/examples/com.eclipsesource.modelserver.example/src/main/java/com/eclipsesource/modelserver/example/ExampleServerLauncher.java) as Java Application, located in the module `com.eclipsesource.modelserver.example`.
```
Comment (edgarmueller) [inline README.md]: as **a** Java Application

File: README.md [hunk 4]
```diff
@@ -1,13 +1,35 @@
...
-
-## Build and run the standalone jar
+To build the modelserver as standalone JAR execute the following maven goal in the root directory:
 ```bash
 mvn clean install -Pfatjar
+```
+
+
+## Run the modelserver
+### Run the modelserver in IDE
```
Comment (edgarmueller) [inline README.md]: **an** IDE

File: README.md [hunk 5]
```diff
@@ -1,13 +1,35 @@
...
 cd  examples/com.eclipsesource.modelserver.example/target/
 java -jar com.eclipsesource.modelserver.example-X.X.X-SNAPSHOT-standalone.jar
 ```
+
+### Modelserver port
+For running the modelserver the default port is `8081`.
+To change the port, use the command line argument `--port=XXXX`.
+
+## Use the modelserver API
+If the modelserver is up and running, you can access the modelserver API via http://localhost:8081/api/model/:modeluri.
```
Comment (edgarmueller) [inline README.md]: Use backticks for URL formatting

File: com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java (hunk 2 above)
Comment (ndoschek) [reply com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerLauncher.java]: Thanks for testing, that's a good point!

**Review by tortmayr — APPROVED** [review]

**Comment by tortmayr** [comment]
Follow-up Issue: #9

**Pull request merged**